app/modules/notifications/models.py

```diff
@@ -120,10 +123,22 @@ def owner(self):
         return self.recipient
 
     def get_sender_name(self):
-        return self.message_values.get('sender_name', 'N/A')
+        from app.modules.users.models import User
+
+        user = User.query.get(self.sender_guid)
+        if user:
+            return user.full_name
+        else:
+            return 'N/A'
```

Comment:
Ha! There was a question yesterday about whether we should standardize on n/a or N/A in houston, lets go with your choice.